include/Utils.h

```diff
@@ -31,6 +28,10 @@ StructType *unwrapValuePointerToStruct(Value *value);
 bool IRstructNameEqualsCstructName(std::string &structName,
                                    std::string &optLoadFileName);
 
+} // namespace Dataflow
+
+namespace Util {
```

Comment:
"Util" is too vague of a name for a namespace, IMO. It's likely to collide with others if we ever want to link this project against another. (This applies to "dataflow" too, to be fair.)

Would you consider namespaces like "rlcdataflow" and "rlcutil"? I think those are a lot less likely to cause collisions in the future, but they're still clear.